autotest/ogr/ogr_mssqlspatial.py

```diff
@@ -245,6 +245,50 @@ def ogr_mssqlspatial_4():
 
     return 'success'
 
+###############################################################################
```

Comment:
hum a lot of code for a single check. Couldn't just the GetFID() check be included in the existing ogr_mssqlspatial_4 test ?

Reply:
I had considered that, but did not know if you preferred to keep separate tests for distinct features/fixes.  I don't mind at all modifying and resubmitting the py file, and I will do both tests in ogr_mssqlspatial_4, returning the reason for failure if either of them fails.

Reply:
Thanks. There's no general policy, depends on the context. But adding 3 lines to an existing test (and soon once the pytest'ification is done, that will be a single assert one-liner) is better for long-term maintenance than adding 40 new lines :-)